main/py-jinja2/APKBUILD

```diff
@@ -3,14 +3,14 @@
 pkgname=py-jinja2
 _pkgname=Jinja2
 pkgver=2.10
-pkgrel=1
+pkgrel=2
 pkgdesc="A small but fast and easy to use stand-alone python template engine"
 url="http://jinja.pocoo.org/"
 arch="noarch"
 license="BSD"
-_py2_depends="py2-markupsafe"
-_py3_depends="py3-markupsafe"
-makedepends="python2-dev python3-dev py-setuptools $_py2_depends $_py3_depends"
+depends="py-markupsafe"
```

Comment:
I am not sure i like this change, which requires that py2/py3-markupsafe are built as sub packages from py-markupsafe. If we for some reason in future split py-markupsafe into 2 separate APKBUILDs, py2-markupsafe and py3-markupsafe, then things will break.

I think its better to explicitly depend on py2/py3-markupsafe and let build servers pull in whatever provides those.

Reply:
ok, i looks like this is according the documentation.